web/src/lib.rs

```diff
@@ -554,6 +558,13 @@ impl Ruffle {
                     .and_then(|q| StageQuality::from_str(&q).ok())
                     .unwrap_or(default_quality),
             )
+            .with_scale(
+                config
+                    .scale
+                    .and_then(|s: String| StageScaleMode::from_str(&s).ok())
```

Comment:
nit: I think the `String` type annotation is not needed here (like it's not needed for `StageQuality` above).